sql/conseil.sql

```diff
@@ -869,3 +869,49 @@ FROM
     AND bitcoin.outputs.n = bitcoin.inputs.v_out
   GROUP BY
     bitcoin.outputs.script_pub_key_addresses;
+
+CREATE SCHEMA ethereum; 
```

Comment:
Since we want to support multiple networks in the same schema, shouldn't we add "network" column as we discussed with Vishakh?

Reply:
I've added `blockchain` column below.

Reply:
Note, we decided to name it "network" with limited options.